x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/constants.ts

```diff
@@ -6,13 +6,11 @@
  */
 
 /**
- * Event merging banner is hidden temporarily for 8.16 (and serverless).
- * Probably will be enabled for 8.17 or 8.18, when we can change the defaults and trigger policy deploy by migration.
- * Blocker issue: https://github.com/elastic/kibana/issues/193352
+ * Enables displaying event merging banner.
  */
-export const ALLOW_SHOWING_EVENT_MERGING_BANNER = false;
+export const ALLOW_SHOWING_EVENT_MERGING_BANNER = true;
```

Comment:
Shouldn't this be provided as a feature flag functionality?

Reply:
nah, looks like unnecessary work. : ) we'll never enable/disable this from kibana config, so what would be the point? it'll be just removed some iterations later.

Reply:
plot twist: in order to keep it disabled on serverless, but be able to enable it even for BC1 from config, it's behind a feature flag now! 1e4b52e9e32b9eca5ab298e71a720bace263d870

Reply:
It's a fast changing world that we live in ;p